src/lib/error_json/error_json.ml

```diff
@@ -181,6 +181,6 @@ let info_to_yojson (info : Info.t) : Yojson.Safe.t =
 let error_to_yojson (err : Error.t) : Yojson.Safe.t =
   match info_to_yojson (err :> Info.t) with
   | `Assoc assocs ->
-      `Assoc (("commit_id", `String Mina_version.commit_id) :: assocs)
+      `Assoc assocs
```

Comment:
Oh no :( This has been incredibly helpful for figuring out what's broken on what version of the code.

Reply:
@mrmr1993 I know, there is a follow-up PR #15865

It specifically targets this issue: for any Warn log (except maybe for logging from unit tests which isn't configured via log consumer/registry) there will be a commit id attached.